src/components/decision/classification/classification.ts

```diff
@@ -1,28 +1,185 @@
+    const shortcuts: KeyboardShortcut[] = [];
+
+    if (this.derivedTrueShortcut) {
+      shortcuts.push({
+        key: this.derivedTrueShortcut,
+        description: `Add a 'true' ${this.tag.text} classification`,
+      });
+    }
+
+    if (this.derivedFalseShortcut) {
+      shortcuts.push({
+        key: this.derivedFalseShortcut,
+        description: `Add a 'false' ${this.tag.text} classification`,
+      });
+    }
+
+    return shortcuts;
+  }
+
+  protected override handleShortcutKey(event: KeyboardEvent): void {
+    if (this.isTrueShortcutKey(event)) {
+      this.trueDecisionButton.click();
+      this.falseDecisionButton.focus();
+      return;
+    }
+
+    if (this.isFalseShortcutKey(event)) {
+      this.falseDecisionButton.click();
+      this.falseDecisionButton.focus();
+    }
+  }
+
+  protected override isShortcutKey(event: KeyboardEvent): Readonly<boolean> {
+    return this.isTrueShortcutKey(event) || this.isFalseShortcutKey(event);
+  }
+
+  private isTrueShortcutKey(event: KeyboardEvent): boolean {
+    if (!this.derivedTrueShortcut) {
+      return false;
+    }
+
+    return event.key === this.derivedTrueShortcut;
+  }
+
+  private isFalseShortcutKey(event: KeyboardEvent): boolean {
+    if (!this.derivedFalseShortcut) {
+      return false;
+    }
+
+    return event.key === this.derivedFalseShortcut;
+  }
+
+  private decisionButtonTemplate(decision: DecisionOptions, decisionId: number): TemplateResult {
+    const buttonClasses = classMap({
+      disabled: !!this.disabled,
+      "oe-btn-primary": decision === DecisionOptions.TRUE || decision === DecisionOptions.FALSE,
+      "oe-btn-secondary": decision === DecisionOptions.UNSURE || decision === DecisionOptions.SKIP,
+    });
+
+    const shortcut = decision === DecisionOptions.TRUE ? this.derivedTrueShortcut : this.derivedFalseShortcut;
+    const decisionModel = new Classification(decision, decisionId, this.tag);
+    const color = this.injector.colorService(decisionModel);
+
+    return html`
+      <button
+        id="${decision}-decision-button"
+        class="decision-button ${buttonClasses}"
+        part="${decision}-decision-button"
+        title="Shortcut: ${shortcut}"
+        style="--ripple-color: var(${color})"
+        aria-disabled="${this.disabled}"
+        @click="${() => this.emitDecision([decisionModel])}"
+      >
+        <span class="oe-pill decision-color-pill" style="background-color: var(${color})"></span>
+
+        <div class="button-text">${decision}</div>
+        ${this.selectionMode !== "tablet" && shortcut ? html`<kbd class="shortcut-legend">${shortcut}</kbd>` : nothing}
+      </button>
+    `;
+  }
+
+  public override render() {
+    const buttonOptions = [DecisionOptions.TRUE, DecisionOptions.FALSE];
+
+    // prettier-ignore
+    return html`
+      <div class="decision-group">
+        <div class="decision-group-title">
+          ${this.tag.text ?? this.tag}
+          <slot></slot>
```

Comment:
todo for normalizing the tag model. Should not need to fallback to outputting `this.tag`.

We should also move the text into the slot, so if the slot is customized by the user it overrides the text.